components/dashboard/src/data/workspaces/create-workspace-mutation.ts

```diff
@@ -22,16 +22,6 @@ export const useCreateWorkspaceMutation = () => {
         onError: (error) => {
             setIsStarting(false);
         },
-        onSuccess: (result) => {
-            if (result && result.createdWorkspaceId) {
-                // successfully started a workspace, wait a bit before we allow to start another one
-                setTimeout(() => {
-                    setIsStarting(false);
```

Comment:
Seems like `isStarting` will remain `true` now even after it's done. Maybe we can just use the isLoading state now instead - `isStarting: mutation.isLoading`?